src/client/ImportClient.ts

```diff
@@ -1,8 +1,6 @@
 import type { BodyOf } from './internal/schema.utils.js';
-
-import FormData from 'form-data';
```

Comment:
I'm not sure if this is clever, there might be still users on the older node.js and this will break it for them.

Reply:
The CLI specifies Node 18 as its engine requirement, and I think so does some of the existing dependencies. I based my decision on that 🤔

Reply:
Ok, if it runs in 18, than it should be fine.